packages/react/src/components/TearSheet/TearSheetWrapper.jsx

```diff
@@ -48,6 +49,32 @@ const TearSheetWrapper = ({ isOpen, className, onCloseAllTearSheets, children })
     )
   );
 
+  const tearSheetsNodes = containers.reduce(
+    (acc, curr) => [...acc, curr?.current?.children[0]],
+    []
+  );
+
+  useEffect(() => {
+    if (
+      activeTearSheetIdx !== null &&
+      !Number.isNaN(
+        Math.round(
```

Comment:
Maybe `Math.round(tearSheetsNodes?.[activeTearSheetIdx]?.children?.[0].getBoundingClientRect().height)` could be a variable.

Reply:
Good catch. Thanks!